answer_10.rb

```diff
@@ -0,0 +1,9 @@
+require 'twitter'
+client = Twitter::REST::Client.new do |config|
+  config.consumer_key        = "4Q2wVbM6q9Tt5YLGy3D9D8RVV"
```

Comment:
@juliatjoy You may want to remove the consumer key and consumer secret which you used for twitter from git. It can be misused if it gets into wrong hands.